diff --git a/src/thpool.c b/src/thpool.c
index 0ecf093..ed57402 100644
--- a/src/thpool.c
+++ b/src/thpool.c
@@ -90,7 +90,7 @@ typedef struct thpool_{
 
 static void  thread_init(thpool_* thpool_p, struct thread** thread_p, int id);
 static void* thread_do(struct thread* thread_p);
-static void  thread_hold();
+static void  thread_hold(int);
 static void  thread_destroy(struct thread* thread_p);
 
 static int   jobqueue_init(thpool_* thpool_p);
@@ -313,7 +313,7 @@ static void thread_init (thpool_* thpool_p, struct thread** thread_p, int id){
 
 
 /* Sets the calling thread on hold */
-static void thread_hold () {
+static void thread_hold (int) {
 	threads_on_hold = 1;
 	while (threads_on_hold){
 		sleep(1);
